src/config/env.js

```diff
@@ -8,11 +8,11 @@ if (ENV === 'development') dotenv.load({
 const config = {
     NODE_ENV: process.env.NODE_ENV,
     PORT: process.env.PORT,
-    DB_URL: process.env.DB_URL
+    DB_URL: process.env.DB_URL,
+    SESSION_SECRET: process.env.SESSION_SECRET
```

Comment:
*The express-session module will use a cookie-stored, signed identifier to identify the current user. To sign the session identifier, it will use a secret string, which will help prevent malicious session tampering. For security reasons, it is recommended that the cookie secret be different for each environment, which means this would be an appropriate place to use our environment configuration file.* - **MEAN Web Development**

The key could be `galletita` if you want to. Is up to your imagination.